.circleci/config.yml

```diff
@@ -174,19 +168,14 @@ jobs:
             gotestsum \
               --format=short-verbose \
               --junitfile $TEST_RESULTS_DIR/gotestsum-report.xml -- \
-              -tags=$GOTAGS -cover -coverprofile=cov_sdk.part \
+              -tags=$GOTAGS -cover -coverprofile=coverage.txt \
               ./...
 
-      # save coverage report parts
-      - persist_to_workspace:
-          root: ./sdk
-          paths:
-            - cov_*.part
-
       - store_test_results:
           path: *TEST_RESULTS_DIR
       - store_artifacts:
           path: *TEST_RESULTS_DIR
+      - run: *codecov_upload
 
   # combine code coverage results from the parallel circleci executors
   coverage-merge:
```

Comment:
Can you remove this whole job now that we don't need it?

Reply:
Also can you add back  `go tool cover -html=coverage.out -o $TEST_RESULTS_DIR/coverage.html`? I think this is helpful if you don't want to use codecov but just click on the HTML in the CircleCI artifacts to get the normal Go built in HTML explorer of coverage

Reply:
Good catch, I forgot to remove the `coverage-merge` job. Did that with an amend.

Now that we upload from 6 different tasks the html report probably isn't that useful. I suspect the html report is not used all that often, and when it is , we could probably generate it just as easily by running the tests locally.